src/main/resources/docker/client/Dockerfile

```diff
@@ -1,60 +1,35 @@
-#
-#  Copyrights     : CNRS
-#  Author         : Oleg Lodygensky
-#  Acknowledgment : XtremWeb-HEP is based on XtremWeb 1.8.0 by inria : http://www.xtremweb.net/
-#  Web            : http://www.xtremweb-hep.org
-#   
-#  This file is part of XtremWeb-HEP.
-# 
-#  XtremWeb-HEP is free software: you can redistribute it and/or modify
-#  it under the terms of the GNU General Public License as published by
-#  the Free Software Foundation, either version 3 of the License, or
-#  (at your option) any later version.
-# 
-#  XtremWeb-HEP is distributed in the hope that it will be useful,
-#  but WITHOUT ANY WARRANTY; without even the implied warranty of
-#  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
-#  GNU General Public License for more details.
-# 
-#  You should have received a copy of the GNU General Public License
-#  along with XtremWeb-HEP.  If not, see <http://www.gnu.org/licenses/>.
-#
-#
-
-# Versionning
-#  July 24th, 2017 
-#  0.3
-#    - xwversion : 10.6.0
-#    - using ADD and WORKDIR to ease usage
-#    - reducing packages list to the necessary only to reduce image size
-#
-
 FROM ubuntu:16.04
-MAINTAINER Oleg Lodygensky (oleg.lodygensky@lal.in2p3.fr)
-LABEL "copyrights"="CNRS 2016"
-LABEL "author"="Oleg Lodygensky"
-LABEL version="0.3"
-LABEL description="This creates a Docker image for the XWHEP client"
-
-EXPOSE 4327
+EXPOSE 4321 4322 4323 4324 4327 4328 443
 
-ENV XWVERSION "12.2.2"
+ADD conf/xtremweb.client.conf /xwhep/conf/
+RUN mkdir -p /xwhep/bin
+ADD bin /xwhep/bin
```

Comment:
We shouldn't add all the content of /bin (https://github.com/iExecBlockchainComputing/xtremweb-hep/issues/16), some scripts are templated with sensitive information
(don't know which one(s) so far, sql?)

Reply:
Some clean has been done in /bin, only needed scripts are imported now.